romancal/regtest/test_tweakreg.py

```diff
@@ -81,6 +81,47 @@ def test_tweakreg(
 
     assert rms < 1.3 / np.sqrt(2)
 
+    # check if the Mean Absolute Error is less that 10 milliarcsec  (DMS406)
+    abs_diff = (np.absolute(diff)) / 1.0e-3
+    mean_abs_error = (tweakreg_out.meta.wcs_fit_results.mae) / 1.0e-3
+    assert mean_abs_error < 10.0
+    assert np.max(abs_diff) < 10.0
+    passmsg = "PASS" if np.max(abs_diff) < 10.0 else "FAIL"
+    dms_logger.info(
+        f"DMS406 {passmsg} the Absolute Astrometric Uncertainty of {np.max(abs_diff):5.2f} mas is less that 10 mas."
+    )
+
+    assert tweakreg_out.meta.cal_step.tweakreg == "COMPLETE"
+    # Find the reference catalog used by tweakreg
+    refcat_name = None
+    for entry in tweakreg_out.meta.cal_logs:
+        if "abs_refcat" in entry:
+            log_substring = entry[entry.rfind("abs_refcat") :]
+            refcat_name = log_substring[: log_substring.find("\n")]
+
+    assert "GAIA" in refcat_name
+    passmsg = "PASS" if "GAIA" in refcat_name else "FAIL"
+    dms_logger.info(
+        f"DMS549 MSG: {passmsg}, {refcat_name} used to align data to "
+        "the Gaia astrometric reference frame."
+    )
+
+    # check if the Mean Absolute Error is less that 5 milliarcsec  (DMS549)
+    mean_abs_error = (tweakreg_out.meta.wcs_fit_results.mae) / 1.0e-3
+    assert mean_abs_error < 5.0
+    passmsg = "PASS" if mean_abs_error < 5.0 else "FAIL"
+    dms_logger.info(
+        f"DMS549 {passmsg} the Mean Absolute Error of {mean_abs_error:5.2f} mas is less that 5 mas."
+    )
+    # check that the tweakreg step is marked complete
+    assert tweakreg_out.meta.cal_step.tweakreg == "COMPLETE"
+    passmsg = "PASS" if tweakreg_out.meta.cal_step.tweakreg == "COMPLETE" else "FAIL"
+    dms_logger.info(f"DMS549 {passmsg} the Tweakreg step is compete.")
```

Comment:
```suggestion
```
This is already done above for DMS280. How do they differ?
https://github.com/spacetelescope/romancal/blob/21dcb64a7cfd31a1d4a72c30fda5fb551e076fdd/romancal/regtest/test_tweakreg.py#L45-L49